### PR TITLE
colors match but delayed update in cluster

### PR DESCRIPTION
**Pull request by kirtansakariya**
The colors match for the cluster and bars now. However, there's a slight delay when they get updated. They first start off as black and then change to their appropriate colors. This probably has to do with `d3.json` being asynchronous. Working on trying to fix that now.

### CODE REVIEW

**Comment by kirtansakariya** [comment]
Also addresses issues #314

**Comment by JaimieMurdock** [comment]
I'm having trouble with this pull request.

The bars and clusters are not rendering as the same color.

One test case: I have the bubble view up and there's only one orange circle. I click on it to go to bars view. I would expect the top topic there to be orange (because it's the same one) and to only have one orange bar. There are 3 orange bars and the topic I selected is gray.

Also, the float of the right side topic navigation is not appearing.

**Review by JaimieMurdock — APPROVED** [review]
For some reason these worked this morning. Perhaps I had my install pointing to the wrong location.

File: www/nodes.js [hunk 1]
```diff
@@ -188,16 +188,26 @@ d3.csv(base_url + "cluster.csv", function(error, data) {
...
+          .attr("id", function(d) { return d.k + '_' + d.topic; })
           .attr("r", function(d) { return d.radius; })
           .attr("cx", function(d) { return x(d[xVar]); })
           .attr("cy", function(d) { return y(d[yVar]); })
+          .attr("visibility", 'hidden')
           //.attr("style", function(d) { return "fill:" + d.color +"; fill-opacity: "+ opacity(d.opacity) + ";"; })
-          .style("fill", function(d) { return d.color; })
+          .style("fill", function(d) { 
+            var url = base_url + d.k + '/topics.json';
+            d3.json(url, function(topics) {
```
Comment (JaimieMurdock) [inline www/nodes.js]: recursive call to get all the topics for every single node? I don't know how I missed this during the merge, but this is why there's a delay.

**Pull request merged**